### ISSUE TITLE
connection closed 

### ISSUE THREAD

**Issue by samupere** [haxball/haxball-issues#2111]
hi there. I have a question, every time I enter any server the game
logs me out and puts connection closed a few seconds after entering, I
want to know what I can do because this does not let me play the game
![imagen_2024-01-30_123651946](https://github.com/haxball/haxball-issues/assets/157840981/a9875ed6-e6c8-4c66-a9ee-7edf072b7574)

**Comment by damian18zb** [comment]
Is it for real ANY server?
There is common bug when server kicks you after join it not shows kick reason message.
Have you tried hosting your own room?

**Comment by samupere** [comment]
@damian18zb Yes is on any server, but on the ones I host that does not happen

**Comment by hisham-tareqz** [comment]
[Disable Chrome Flag 'Anonymize local IPs exposed by WebRTC](https://github.com/haxball/haxball-issues/wiki/Connection-Issues#disable-chrome-flag-anonymize-local-ips-exposed-by-webrtc).
